### ISSUE TITLE
bad accessibility

### ISSUE THREAD

**Issue by ahmedSamy31** [microsoft/terminal#6593]
i want to use the new windows termenal in my daly work but it doesn't work with my NVDA screen reader and it give me a bad accessibility errors inshort the terenal like plank page normaly i use git bash as a termenal but doesn't have that mutch feachers  

# Environment

```none
Windows build number: [run `[Environment]::OSVersion` for powershell, or `ver` for cmd]
Windows Terminal version (if applicable):

Any other software?
```

# Steps to reproduce

# Expected behavior

# Actual behavior

**Comment by zadjii-msft** [comment]
You're gonna have to give us a few more details than that - what version of NVDA are you using? What exactly are you trying to do, what do you expect to happen, and what's actually happening? Having specific repro steps is the only way we can help investigate an issue. @carlos-zamora has done a _ton_ of work in the last year to make sure the Terminal supports NVDA and Narrator, so if there's anything wrong, I'm sure he'd be happy to help fix it ☺️

**Comment by carlos-zamora** [comment]
Hi @ahmedSamy31,

We've been working with NVDA. They added support for Windows Terminal in NVDA 2020.2 Beta 1. You can download the updated version at https://www.nvaccess.org/post/nvda-2020-2beta1/.

You might have to enable the following on NVDA:
- Go to Preferences, then Settings
- Select the "Advanced" category 
- Under "Microsoft UI Automation", enable "Use UI Automation to access the Windows Console when available".

Tagging @codeofdusk to verify if I'm missing anything.

Separately, @cinnamon-msft maybe we should add this to our docs page under troubleshooting?

**Comment by codeofdusk** [comment]
> Hi @ahmedSamy31,
> 
> 
> 
> We've been working with NVDA. They added support for Windows Terminal in NVDA 2020.2 Beta 1. You can download the updated version at https://www.nvaccess.org/post/nvda-2020-2beta1/.
> 
> 
> 
> You might have to enable the following on NVDA:
> 
> - Go to Preferences, then Settings
> 
> - Select the "Advanced" category 
> 
> - Under "Microsoft UI Automation", enable "Use UI Automation to access the Windows Console when available".
> 
> 
> 
> Tagging @codeofdusk to verify if I'm missing anything.
> 
> 
> 
> Separately, @cinnamon-msft maybe we should add this to our docs page under troubleshooting?

Nope, no need to enable UIA in console for Windows Terminal – it always uses UIA as the implementation is good. That setting is only for conhost.

**Comment by ahmedSamy31** [comment]
i'm using NVDA2020.1 and i had this satings enabled  the problem is when i type any command NVDA can't read the outbot of this command like when i type "novd -v" to chick node js version it prented on the screen but NVDA doesn't read any thing and  i can't navigate the result or the oll window

**Comment by srisuk2519** [comment]
เมื่อ ส. 20 มิ.ย. 2563 เวลา 02:16 ahmed samy alkhouly <
notifications@github.com> เขียนว่า:

**Comment by zadjii-msft** [comment]
@ahmedSamy31 Can you try NVDA 2020.2 Beta 1 instead of NVDA 2020.1?

**Comment by ahmedSamy31** [comment]
i tried nvda2020.2 and it works realy thank you
